index.html

```diff
@@ -138,8 +158,13 @@ <h4>Multi-Post
                     A daily selection of privately personalized reads; 
                     no accounts or sign-ups required.
                 </p>
+                <p class="Info2">
+                    A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
```

Comment:
- [ ] please make the text section of your cards responsive when switching from the mobile version to the desktop version or vice versa. 
your version.
![Screenshot (101)_LI](https://user-images.githubusercontent.com/65298048/171615613-e0d21ce9-fcfe-42b3-81da-7cec6f43d90f.jpg)